src/parser/parse_tree_cl.c

```diff
@@ -6392,10 +6394,7 @@ pt_apply_alter_trigger (PARSER_CONTEXT * parser, PT_NODE * p, PT_NODE_FUNCTION g
 static PT_NODE *
 pt_init_alter_trigger (PT_NODE * p)
 {
-  p->info.alter_trigger.trigger_owner = NULL;
-  p->info.alter_trigger.trigger_priority = NULL;
-  p->info.alter_trigger.trigger_spec_list = NULL;
-  p->info.alter_trigger.comment = NULL;
+  memset (&(p->info.alter_trigger), 0x00, sizeof (p->info.alter_trigger));
```

Comment:
Instead of writing the same code for each `init_*` function, how about applying `memset()` for `PT_STATEMENT_INFO union`? (Specifically, `p->info`)

Reply:
`memset (node->info, 0x00, sizeof (p->info));` could be added before `(pt_init_f[node_type]) (node);` in `pt_init_node()` function.

Reply:
Not all init_*() functions were modified, so it was not possible to perform memset() in batches in the pt_init_node() function.
Instead, it might be possible to do memset(node->info) inside each init_*() function.
However, it is not sensitive, so I will try to keep it as it is.